joomla/joomla-cms
# Add basic support for sass/scss files to Template manager editing

#### Summary of Changes

Add ability to see sass files in the list of available files in the tree
Add ability to create a new file of type sass/scss
Add ability to open, edit and save a sass/scss file
#### Testing Instructions

Install patch and go to template manager and select a template (not a style)
Select New File from the toolbar
Create a new file and select sass from the file type dropdown
Check that file can be seen in the folder tree and can be selected, edited, saved and deleted
